manual/deploy/https_with_nginx.md

```diff
@@ -1,78 +1,101 @@
 # Enabling Https with Nginx
 
-Here we suggest you use [Let’s Encrypt](https://letsencrypt.org/getting-started/) to get a certificate from a Certificate Authority (CA). If you use a paid ssl certificate from some authority, just skip the first step.
+When using HTTPS, traffic from and to Seafile Server is encrypted. HTTPS requires a SSL certificate from a Certificate Authority (CA).
 
-### Generate SSL certificate
+For production use, HTTPS is imperative. 
 
-For users who use Let’s Encrypt, you can obtain a valid certificate via [Certbot ACME client](https://certbot.eff.org/)
+Unless you already have a SSL certificate, we recommend that you get your SSL certificate from [Let’s Encrypt](https://letsencrypt.org/). If you have a SSL certificate from another CA, skip the section "Getting a Let's Encrypt certificate".
 
-On Ubuntu systems, the Certbot team maintains a PPA. Once you add it to your list of repositories all you'll need to do is apt-get the following packages.
+## Setup
 
-```bash
-sudo apt-get update
-sudo apt-get install software-properties-common
-sudo add-apt-repository ppa:certbot/certbot
-sudo apt-get update
-sudo apt-get install python-certbot-nginx
-```
+The configuration of Seafile behind Nginx as a reverse proxy is demonstrated using the sample host name `seafile.example.com`. 
 
-Certbot has an Nginx plugin, which is supported on many platforms, and automates both obtaining and installing certs:
+These instructions assume the following requirements:
 
-```bash
-sudo certbot --nginx
-```
+* Seafile Server Community Edition/Professional Edition and [Nginx](deploy_with_nginx.md) were set up according to the instructions in this manual
+* A host name points at the IP address of the server and the server is available on port 80 and 443
+
+If your setup differs from thes requirements, adjust the following instructions accordingly.
+
+### Getting a Let's Encrypt certificate
+
+Getting a Let's Encrypt certificate is straightforward thanks to [Certbot](https://certbot.eff.org/). Certbot is a free, open source software tool for requesting, receiving, and renewing Let's Encrypt certificates.
+
+First, go to the [Certbot](https://certbot.eff.org/) website and choose your webserver and OS.
+![grafik](https://user-images.githubusercontent.com/41058728/126880778-ed2e8f13-c781-4448-8803-4919a380ac7c.png)
```

Comment:
Images files should be uploaded to a folder in this repository and use a relative path for reference. After the website is generated, it will use the correct path.

Reply:
Understood! That makes sense.

Can I upload files? This is what I see when I try to upload images.

![grafik](https://user-images.githubusercontent.com/41058728/126968759-375f2028-3b83-4a29-8040-e7b0c53392bb.png)